client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,18 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
+		return (
+			<EmptyContent
+				title={ this.props.translate( 'Sorry, themes upload is not yet available for multi-site installations.' ) }
```

Comment:
I don't remember where I saw it, but I believe we shouldn't apologize for missing features, or lack of functionality. I think when something goes wrong, it's ok though. Wish I knew where I read that!